openmdao/docs/openmdao_book/features/core_features/working_with_components/distributed_components.ipynb

```diff
@@ -39,9 +39,11 @@
    "source": [
```

Comment:
<p>The output is actually distributed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2390/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>In the code you use the <code>_dup</code>  extension to indicate "non-distributed" without actually introducing the term "duplicated" in the text as far as I  can see.</p><p> </p><p>I wonder if "duplicated" would be a better term in general than "non-distributed"...</p>

Reply:
@swryan : We actually changed these all to "duplicated" at first, but found it kind of confusing terminology, particularly in any error messages, where the user might think they had a duplicate input.  We settled on non-distributed since it is simple and accurate, though a little wordy.  I just forgot to change the varname extension to something else.  Maybe just _nd will do.

Reply:
ah ok.. maybe "replicated" is better.. I personally find it more descriptive/meaningful ¯\_(ツ)_/¯

Reply:
i like "replicated" too ... i think this is something we can ask the users. I'll send out a query and get some feedback on it

Reply:
I've renamed variables with suffix `_dup` to `_nd`.

Reply:
to close the loop on the discussion, the deciding factor was that `non-distributed` more closely matches the API (i.e. distributed=True distributed=False)